index.js

```diff
@@ -10,9 +10,39 @@ function parseGitUri(uri) {
     uri.match(/git@github.com:(.*)\/(.*)\.git/);
 }
 
+function verifyChangelog() {
+  var changelog = exec('git diff HEAD~1');
+  var lines = changelog.split('\n');
+  var isOneLine = (lines.length === 1);
```

Comment:
Can you double check this line? A one-line file would probably return 2 here. A typical one-line file would look something like `This is the only line\n`, which splits as `[ 'This is the only line', '']`.

Maybe we can pop off `lines[lines.length-1]` if it's empty.